ubuntu-agent-vm/scripts/netcoresdk3-1.sh

```diff
@@ -7,3 +7,5 @@ apt-get update
 apt-get install -y apt-transport-https
 apt-get update
 apt-get install -y dotnet-sdk-3.1
+
+export DOTNET_CLI_HOME="/tmp/DOTNET_CLI_HOME"
```

Comment:
```suggestion
export DOTNET_CLI_HOME="/tmp/dotnet"
```
It doesn't have to be DOTNET_CLI_HOME, it can just be dotnet. It looks prettier.
https://blog.jongallant.com/2018/09/solution-users-home-directory-could-not-be-determined/